### PR TITLE
add scenario for handling tracking w/o CCC

### PR DESCRIPTION
**Pull request by mtosi**
I created a new scenario for handling the track reconstruction w/o CCC
this is mainly meant for having the possibility of running the tracking w/o CCC on dedicated streams/PDs in case of need @tier0

hope it is fine

NB:
it should be back ported in 80x, if we want to make use of it in 2016 data processing
please, let me know if I can do a dedicated PR for 80x

NB:
the implementation is closer to a trick, rather than a proper handling of each instances of the cut w/in the reco config,
but I was not able to handle all different instances :(

ps:
thanks to @slava77 and @mmusich for the suggestions !

@rovere @VinInn @mmusich @boudoul @dimattia

### CODE REVIEW

**Comment by mmusich** [comment]
please test

File: Configuration/DataProcessing/python/Impl/trackingnoccc.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    """
+    def promptReco(self, globalTag, **args):
+        """
+        _promptReco_
+
+        Proton collision data taking prompt reco
+
+        """
+
+#        if not 'skims' in args:
```
Comment (slava77) [inline Configuration/DataProcessing/python/Impl/trackingnoccc.py]: is this needed?
please clean up commented out parts or add some comments why these may be needed for the future

File: RecoTracker/Configuration/python/customiseForRunIInoCCC.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+def customiseForRunIInoCCC(process):
+
+    # overwrite parameters which handles the CCC value            
+    if hasattr(process,'SiStripClusterChargeCutTiny'):
+        setattr(process,'SiStripClusterChargeCutTiny', cms.PSet(value = cms.double( -1.0 ) ) )  #  800.0
```
Comment (slava77) [inline RecoTracker/Configuration/python/customiseForRunIInoCCC.py]: why not simply ``` process.SiStripClusterChargeCutTight.value = -1. ``` ?

**Comment by slava77** [comment]
@mtosi @mmusich 
we looked at this with Carl and there are problems: the scenario will not run as is due to definitely present dependencies downstream.
The pp scenario (base for trackingnoccc) includes EI step (event interpretation), which expects PF etc, while RECO:reconstruction_trackingOnly does not run PF etc.

We could fix it in a follow up PR or in this one, but maybe we should first understand which settings this trackingnoccc will have while running on the specified sample.
- which dataset is this scenario going to run on? IIRC, at T0 there is support only one scenario possible per dataset. Just to be sure, this extra dataset will not need regular pp processing, right?
- which output data tiers will be produced
- which DQM, AlCa and Skim sequences will be running?
Depending on the needs, it may be simpler just to run full reco.
My goal is to get a somewhat realistic runable configuration with this PR instead of putting a placeholder that needs fixes later.

**Comment by mmusich** [comment]
@slava77, I think Mia can explain here better the needs of the TRK POG. 
On the AlCa side what is needed is a scenario to reconstruct data (without CCC) from a separate event stream to be used to calibrate the Strip APV particle gains. 
The output data tier would be ALCARECO and ALCAPROMPT, though I am not sure Tier-0 can support to reconstruct at "Express" anything else than the standard express stream.
For AlCa we don't need PF at all. Tracks and clusters is all is needed.
I would suggest on the other hand to discuss the details of this in another forum than a gitHub chat. What meeting would be suitable?

**Comment by slava77** [comment]
The earlier suggestion was the reco meeting. It's Wednesday at 17:30.
We have probably accumulated quorum by now.

**Comment by slava77** [comment]
It looks like this PR can be shelved for now, until the discussions converge.

**Comment by mtosi** [comment]
as discussed yesterday w/ tracker DPG, strip calibration and tracking POG
there is not need of tracking w/o CCC, now
=> I close this PR

**Pull request closed without merging**